protos/repository.proto

```diff
@@ -36,48 +36,55 @@ message BentoMetadata {
     string docs = 3;
   }
 
-  string created_at = 1;
-  string name = 2;
-  string version = 3;
-  BentoServiceEnv env = 4;
-  repeated BentoArtifact artifacts = 5;
-  repeated BentoServiceApi apis = 6;
-  BentoUri uri = 7;
+  BentoUri uri = 1;
+  string created_at = 2;
+  string name = 3;
+  string version = 4;
+  BentoServiceEnv env = 5;
+  repeated BentoArtifact artifacts = 6;
+  repeated BentoServiceApi apis = 7;
 }
 
-// Requests
-message RemoveBentoRequest {
-  string bento_name = 1;
-  string bento_version = 2;
+message AddBentoRequest {
+  BentoUri uri = 1;
 }
 
-message GetBentoRequest {
-  string bento_name = 1;
-  string bento_version = 2;
+message AddBentoResponse {
+  Status status = 1;
+  BentoMetadata bento = 2;
 }
 
-message ListBentoRequest {
+message UploadBentoResponse {
+  Status status = 1;
+  BentoMetadata bento = 2;
+}
+
+message DangerouslyDeleteBentoRequest {
   string bento_name = 1;
-  int32 offset = 2;
-  int32 limit = 3;
+  string bento_version = 2;
 }
 
-// Responses
-message AddBentoResponse {
+message DangerouslyDeleteBentoResponse {
   Status status = 1;
   BentoMetadata bento = 2;
 }
 
-message RemoveBentoResponse {
-  Status status = 1;
-  BentoMetadata bento = 2;
+message GetBentoRequest {
+  string bento_name = 1;
+  string bento_version = 2;
 }
 
 message GetBentoResponse {
   Status status = 1;
   BentoMetadata bento = 2;
 }
 
+message ListBentoRequest {
```

Comment:
I get this is listing all version with the same bento name.

Do we have another request that list all bento in the store?  It might be useful to add.